frontend-ssr/components/cli_git/GitCli.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables';
+
+.commands {
+    width: 35vw;
```

Comment:
The best practice here is not to give the component a width, but to get it's width from it's parent, because we don't know where it will be used. So, let's give it 100% width, and restrict the parent size to `35vw`.